playground/backend/internal/setup_tools/life_cycle/life_cycle_setuper.go

```diff
@@ -58,3 +75,70 @@ func Setup(sdk pb.Sdk, code string, pipelineId uuid.UUID, workingDir string, pre
 	}
 	return lc, nil
 }
+
+// prepareGoFiles prepares file for Go environment.
+// Copy go.mod and go.sum file from /path/to/preparedModDir to /path/to/workingDir.
+func prepareGoFiles(lc *fs_tool.LifeCycle, preparedModDir, workingDir string, pipelineId uuid.UUID) error {
```

Comment:
How is it related to saving java logs to the cache?

Reply:
The previous version of `CopyFiles()` had some additional logic instead of simply copying files. I changed this method so after that I also need make some refactoring with methods that use old `CopyFiles()`